config.toml

```diff
@@ -18,4 +18,4 @@ better software for a better world."""
 [params.contact]
 
 org   = "Willow Bark Co-op"
-email = "hello@example.net"
```

Comment:
Unrelated change, can this be removed from this PR? (although, "oops, good catch, I thought this was fixed already!")

Reply:
sure it's now in #19